.dockerignore

```diff
@@ -46,4 +55,26 @@ output_iso/
 include/utils/stopevents_defs.h
 include/utils/stopevents_data.h
 orioledb.typedefs
-!**/ci/antithesis/libvoidstar.so
\ No newline at end of file
+!**/ci/antithesis/libvoidstar.so
```

Comment:
Sorry, it was copied from .gitignore not in this request, but it's probably better to replace it with just `ci/antithesis` (This is not needed for docker image files)